File: teamcity/src/test/java/BaseTest.java
import com.codeborne.selenide.Configuration;
import org.junit.jupiter.api.BeforeAll;

public class BaseTest {
    protected static final String serverUrl = "http://localhost:8111";

    @BeforeAll
    public static void setup() {
        Configuration.baseUrl = serverUrl;
    }
}

File: parabank/src/test/kotlin/com/parasoft/parabank/automated/BaseTest.kt
package com.parasoft.parabank.automated

import com.codeborne.selenide.Configuration
import org.junit.jupiter.api.BeforeAll

open class BaseTest {
    companion object {
        @JvmStatic
        @BeforeAll
        fun selenideSetup(): Unit {
            Configuration.baseUrl = "https://parabank.parasoft.com"
            Configuration.timeout = 10_000
        }
    }
}